mobile/src/main/java/com/ataulm/wutson/discover/GenresPagerAdapter.java

```diff
@@ -10,14 +12,14 @@
 import java.util.Collections;
 import java.util.List;
 
-class DiscoverByGenrePagerAdapter extends PagerAdapter {
+class GenresPagerAdapter extends PagerAdapter {
```

Comment:
reduce unnecessary name prefixing of classes that are package protected anyway - this is in the discover package